packages/charts/src/SeverityLine/SeverityLine.js

```diff
@@ -45,9 +45,9 @@ const SeverityLine = ({ title, value, className, tooltipMessage, config, chartPr
 
 SeverityLine.propTypes = {
     value: PropTypes.number.isRequired,
-    title: PropTypes.string.isRequired,
+    title: PropTypes.oneOfType([ PropTypes.node, PropTypes.arrayOf(PropTypes.node) ]).isRequired,
```

Comment:
Isn't PropTypes.node the same as an array of nodes?

Reply:
Yes, it is. Nasty habit of mine :+1: